release-notes/topics/aap-23.adoc

```diff
@@ -0,0 +1,30 @@
+[[aap-2.3-intro]]
+= Ansible Automation Platform 2.3
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+.Enhancements
+
+* Fixed a race condition where the UI would not properly populate upon launch
+* Fixed an issue where puppet managed files are not handled properly by the set-up script.
+* Fixed an issue where self-signed certs were being recreated on every run of `setup.sh`.
+* Added an option for execution environment images to be pulled from Hub only.
+* Fixed an issue where the bundled installer was failing when DNF was trying to fetch gpg keys remotely.
+* Upgraded pulp_installer to 3.20.5+ .
+* Implemented sidecar documentation to allow easy documentation of filter and test plugins, as well as documentation for non-python modules without requiring a .py file for documentation.
+* Migrated display for `stdout` and `stderr` from the display class to proxy over the queue for dispatch in the main process to improve reliability of displaying information to the terminal.
+* Moved handler processing into the configured strategy, so that handlers operate within the configured strategy, instead of using a non-configurable linear like execution of handlers.
+* Migrated display for `stdout` and `stderr` from the display class to proxy over the queue for dispatch in the main process to improve reliability of displaying information to the terminal.
+* Moved handler processing into the configured strategy, so that handlers operate within the configured strategy, instead of using a non-configurable linear like execution of handlers
```

Comment:
This looks like a duplicate of line 16 and can be removed.

Reply:
Thanks for catching this!